arduino_mkrzero/src/lib.rs

```diff
@@ -0,0 +1,130 @@
+#![no_std]
+
+extern crate atsamd21_hal as hal;
+
+#[cfg(feature = "rt")]
+extern crate cortex_m_rt;
+#[cfg(feature = "rt")]
+pub use cortex_m_rt::entry;
```

Comment:
+1 on this.  I was a little disappointed by the amount of boiler plate that the executables needed with the changes around `entry`.